tools/PI/DevHome.PI/Program.cs

```diff
@@ -239,7 +239,7 @@ private static void OnActivated(object? sender, Microsoft.Windows.AppLifecycle.A
                             barWindow.NavigateTo(pageType);
                         }
 
-                        barWindow.ShouldExpandLargeWindow = true;
+                        // barWindow.ShouldExpandLargeWindow = true;

```

Comment:
>// barWindow.ShouldExpandLargeWindow = true; [](http://example.com/codeflow?start=24&length=44)

Is it not required anymore?

Reply:
Ended up updating the code to always expand the window if you navigate to a page.